iree/compiler/Dialect/HAL/Target/CUDA/CUDATarget.cpp

```diff
@@ -41,6 +41,10 @@ static llvm::cl::opt<bool> clDisableLoopNounrollWa(
         "Disable the workaround for bug in ptxas for CUDA version before 11.4"),
     llvm::cl::init(false));
 
+static llvm::cl::opt<std::string> clTargetChip(
+    "iree-cuda-llvm-target-arch", llvm::cl::desc("LLVM target chip"),
```

Comment:
Nit: the arch number is not llvm specific I think? Why not just call it `iree-cuda-target-arch`?

Reply:
well it is meant to match llvm option for `-march`. I believe we will want more options matching LLVM target machine in the future and I think it makes sense to keep it in sync with llvm.